src/Compilers/CSharp/Portable/Symbols/Retargeting/RetargetingNamedTypeSymbol.cs

```diff
@@ -412,5 +412,10 @@ internal override IEnumerable<(MethodSymbol Body, MethodSymbol Implemented)> Syn
                 }
             }
         }
+
+        internal override bool HasInlineArrayAttribute(out int length)
+        {
+            return _underlyingType.HasInlineArrayAttribute(out length);
```

Comment:
Do I understand correctly that RetargetingNamedTypeSymbol is used for type forwarding? Should we have a test verifying that type forwarding works with inline array struct definitions?

Reply:
> Do I understand correctly that RetargetingNamedTypeSymbol is used for type forwarding?

No. Retargeting symbols have no primary relation with type forwarding.

> Should we have a test verifying that type forwarding works with inline array struct definitions?

Given trivial implementation, the only reason to have a test is completeness, I think.

Reply:
Added ```InlineArrayType_18_Retargeting``` test